client/package.json

```diff
@@ -3,43 +3,43 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "npm run ng serve -- --aot --port 4201 -c local",
+    "start": "npm run ng serve -- --port 4201 -c local",
     "build": "ng build -c production",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "12.1.1",
+    "@angular/animations": "13.1.0",
     "@angular/cdk": "12.1.1",
-    "@angular/common": "12.1.1",
-    "@angular/compiler": "12.1.1",
-    "@angular/core": "12.1.1",
-    "@angular/forms": "12.1.1",
-    "@angular/localize": "12.1.1",
-    "@angular/platform-browser": "12.1.1",
-    "@angular/platform-browser-dynamic": "12.1.1",
-    "@angular/router": "12.1.1",
-    "@datorama/akita": "5.2.3",
-    "@fortawesome/angular-fontawesome": "0.7.0",
-    "@fortawesome/fontawesome-svg-core": "1.2.30",
-    "@fortawesome/free-solid-svg-icons": "5.14.0",
+    "@angular/common": "13.1.0",
+    "@angular/compiler": "13.1.0",
+    "@angular/core": "13.1.0",
+    "@angular/forms": "13.1.0",
+    "@angular/localize": "13.1.0",
+    "@angular/platform-browser": "13.1.0",
+    "@angular/platform-browser-dynamic": "13.1.0",
+    "@angular/router": "13.1.0",
+    "@datorama/akita": "7.0.1",
+    "@fortawesome/angular-fontawesome": "0.10.1",
+    "@fortawesome/fontawesome-svg-core": "1.2.36",
+    "@fortawesome/free-solid-svg-icons": "5.15.4",
     "@microsoft/signalr": "3.1.8",
-    "@ng-bootstrap/ng-bootstrap": "7.0.0",
-    "bootstrap": "4.5.2",
+    "@ng-bootstrap/ng-bootstrap": "11.0.0-rc.0",
```

Comment:
Looks like 11 was fully released
https://github.com/ng-bootstrap/ng-bootstrap/releases/tag/11.0.0

Reply:
Looks like 12 will support Bootstrap 5 + Angular 13

https://github.com/ng-bootstrap/ng-bootstrap/releases/tag/12.0.0-beta.4

Reply:
updated to release version!

Reply:
i'll have to look into bootstrap 5 and what that offers over v4.